packages/@aws-cdk/aws-ec2/lib/peer.ts

```diff
@@ -77,6 +77,16 @@ class CidrIPv4 implements IPeer {
   public readonly uniqueId: string;
 
   constructor(private readonly cidrIp: string) {
+    const cidrMatch = cidrIp.match(/^(\d{1,3}\.){3}\d{1,3}(\/\d+)?$/);
+
+    if (!cidrMatch) {
+      throw new Error(`Invalid IPv4 CIDR: "${cidrIp}"`);
+    }
+
+    if (!cidrMatch[2]) {
+      throw new Error(`CIDR mask is missing in IPv4: "${cidrIp}". Did you mean "${cidrIp}/xx"?`);
```

Comment:
I feel it makes more sense to replace `/xx` with `/32` in the error message. If people wrote a single IP address, that's probably what they meant, and if they didn't it's clear enough where to substitute values to make it do what they want.

I considered automatically adding `/32` to the IP address, but we should probably make a separate class for single IP addresses.

Reply:
> we should probably make a separate class for single IP addresses.

I like that idea. We could also deprecate `ipv4/v6` and explicitly name them `ipv4Cidr`

Reply:
Otherwise, done